### PR TITLE
add the author block in the bottom

### PR DESCRIPTION
**Pull request by garfieldduck**
added the author block in the bottom of the article page

### CODE REVIEW

File: src/components/article/BottomAuthor.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    super(props)
+  }
+
+  setHtml(html) {
+    return { __html: html }
+  }
+
+  render() {
+    const { authors } = this.props
+    console.log("AUTHOR", authors)
```
Comment (nickhsine) [inline src/components/article/BottomAuthor.js]: remove console?

File: src/themes/bootstrap.overrides.scss [hunk 2]
```diff
@@ -1,3 +1,12 @@
+
+.container {
+  line-height: 1.8;
+  
+  @media only screen and (min-width: $screen-md-min) {
```
Comment (nickhsine) [inline src/themes/bootstrap.overrides.scss]: where does $screen-md-min come from?

File: src/components/article/BottomAuthor.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
+@import '../../themes/variables';
```
Comment (nickhsine) [inline src/components/article/BottomAuthor.scss]: do we need this import ?

File: src/components/article/BottomAuthor.js (hunk 1 above)
Comment (garfieldduck) [inline src/components/article/BottomAuthor.js]: thanks. I removed it.

File: src/themes/bootstrap.overrides.scss (hunk 2 above)
Comment (garfieldduck) [inline src/themes/bootstrap.overrides.scss]: It's a bootstrap-sass's variable

File: src/components/article/BottomAuthor.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    const authorRows = _.map(authors, author =>
+      <div className={'row ' + styles.authorRow}>
+        <div className="col-sm-2">
```
Comment (nickhsine) [inline src/components/article/BottomAuthor.js]: should we use ```col-md-*``` ? 
Do we probably have ```tablet``` version mockup?

File: src/components/article/BottomAuthor.js [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+          <img src="https://ssl.gstatic.com/accounts/ui/avatar_2x.png" className="img-responsive img-circle center-block"/>
+        </div>
+        <div className="col-sm-10">
+          <p>|<span className={styles.authorName}>{author.name}</span>|</p>
+          {author.bio && author.bio.html ?
```
Comment (nickhsine) [inline src/components/article/BottomAuthor.js]: ```_.get(author, ['bio', 'html'])``` ?

File: src/components/article/BottomAuthor.js (hunk 4 above)
Comment (garfieldduck) [inline src/components/article/BottomAuthor.js]: col-sm-* supports screens larger than tablets, while col-md-* only supports screens larger than desktop

File: src/components/article/BottomAuthor.scss (hunk 3 above)
Comment (garfieldduck) [inline src/components/article/BottomAuthor.scss]: I will remove it. thanks!

**Pull request merged**